service/matching/physical_task_queue_manager.go

```diff
@@ -416,7 +417,17 @@ func (c *physicalTaskQueueManagerImpl) LegacyDescribeTaskQueue(includeTaskQueueS
 
 func (c *physicalTaskQueueManagerImpl) Describe() *taskqueuespb.PhysicalTaskQueueInfo {
 	return &taskqueuespb.PhysicalTaskQueueInfo{
-		Pollers: c.GetAllPollerInfo(),
+		Pollers:     c.GetAllPollerInfo(),
+		BacklogInfo: c.GetBacklogInfo(),
+	}
+}
+
+func (c *physicalTaskQueueManagerImpl) GetBacklogInfo() *taskqueuepb.BacklogInfo {
+	return &taskqueuepb.BacklogInfo{
+		ApproximateBacklogCount: c.backlogMgr.db.getApproximateBacklogCount(),
+		ApproximateBacklogAge:   nil,        // TODO: Shivam - add this feature
```

Comment:
These three features are expected to be added after the backlog counter works as expected.